phalcon/mvc/model/query.zep

```diff
@@ -3325,7 +3325,7 @@ class Query implements QueryInterface, InjectionAwareInterface
 			}
 
 			let result = cache->get(key, lifetime);
-			if result !== null {
+			if !empty result {
```

Comment:
Sorry to be a pain but this needs to be in its own PR since it doesn't relate to this one. This way we will know what changed when.

Reply:
Can we also write a test or two for this PR?

Reply:
I can do that, but I felt it was related as the cache can return "falsey" values. Now that the cache adapters can return `""`, `false` and `null` it seemed like an appropriate "catch-all" because it won't simply return `null` anymore.

I'll get some tests sorted too.

Reply:
Thanks bud